usefulUtilities/webAppTemplate/README.md

```diff
@@ -0,0 +1,50 @@
+# Description
+
+An express template for web apps.  
+
+# Instructions
+
+```
+You will create two config files.  A config JSON for the server with the following fields:
+{
+    "PORT": <Port Number>,
+    "API_ROUTER": "<./pathToYourRoutingFile.j>",
+    "API_PATH": "</endpointAPI/path>"
+}
+```
+
+as well as a path for your database setup
+```
+{
+    "config": {
+        "mySQLHost": "<Hostname of the MySQL server>",
+        "mySQLUsername": "<Username you'll use to connect to the MySQL server>",
+        "mySQLPassword": "<Password you'll use to connect to the MySQL server>",
+        "databaseName": "<Name of the DB you'll use for this webapp>"
+        "multipleStatements": <true/false depending on if you have multiple tables below to setup>
```

Comment:
where is this used? i see it referenced once but not actually used

Reply:
That is mainly used in dbMaker.js.  I think it's turned off by default to prevent against injection attacks. Going to add a note in the readme about that.
    maybeCreateTables() {
        const extendedConfig = Object.assign({}, baseConfig, {
            database: dbInfo.config.databaseName,
            multipleStatements: dbInfo.config.multipleStatements
        })

        return Database.execute(extendedConfig, 
            database => database.query(this.createTableQuery));
    }